tests/functional/sign_up.js

```diff
@@ -233,7 +233,7 @@ define([
         .then(fillOutSignUp(email + '-', PASSWORD))
 
         // wait five seconds to allow any errant navigation to occur
-        .then(noPageTransition(selectors.SIGNUP.HEADER, 5000))
```

Comment:
2000 was too short to catch errors on CI servers, so instead of duplicating 5000 everywhere, I just changed the default to 5000 and removed these.